src/Lykke.Job.BlockchainCashinDetector/Workflow/Sagas/CashinSaga.cs

```diff
@@ -335,7 +336,24 @@ private async Task Handle(OperationExecutionFailedEvent evt, ICommandSender send
                 return;
             }
 
-            var transitionResult = aggregate.OnTransactionFailed(evt.Error);
+            var transitionResult = aggregate.OnTransactionFailed(evt.Error, evt.ErrorCode.MapToCashinErrorCode());
+
+            if (aggregate.ErrorCode.HasValue)
```

Comment:
this filed SHOULD be not null here, so check it for null and throw if it is null. See how it done in another event handlers